test/integration/Pool.spec.ts

```diff
@@ -88,6 +90,12 @@ describe('P2P Pool Tests', async () => {
     expect(nodeReputationPromise.banned).to.be.true;
   });
 
+  it('should throw error when connecting to tor node', async () => {
+    const address = addressUtils.fromString('.onion');
```

Comment:
I think it makes sense to use a real onion address for completeness sake, you can plug in 3g2upl4pq6kufc4m.onion which belongs to DuckDuckGo.

Reply:
thanks)